### PR TITLE
udkommentere i ansat

### PR DESCRIPTION
**Pull request by Gi5lund**
se title

### CODE REVIEW

File: src/Ansat.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import java.time.LocalDate; //Import af localtime for at kunne sætte datoer.
 import java.util.ArrayList;
 public class Ansat {
-    Texthandler th = new Texthandler();
+   // Texthandler th = new Texthandler();
```
Comment (MKaae) [inline src/Ansat.java:5]: I cannot approve of this as there is a outcommented section that is needed for the code. Please reconsider this action.

**Review by MKaae — CHANGES_REQUESTED** [review]
Please review the added comment to the top of the code that you have submitted.

**Comment by Gi5lund** [comment]
have reconsidered. you're right and change has been reversed.

**Review by MKaae — APPROVED** [review]
This section of code is approved by the King. Your welcome to merge into the master branch.

**Pull request merged**